src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -69,6 +70,7 @@ public class ClusterTransactionOptions : IDynamicJson
             public TimeSpan? WaitForIndexesTimeout;
             public bool WaitForIndexThrow;
             public List<string> SpecifiedIndexesQueryString;
+            public bool DisableAtomicDocumentWrites;
```

Comment:
This will be false for legacy commands? Is this OK?

Reply:
I don't think that this is a scenario that we need to worry about.